package org.revature.p1.utils.exceptions;

public class AttemptUpdatingFinalizedTicketException extends RuntimeException {
}
